server/middleware/routes-acl/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function(options) {
+  options = options || {};
+  console.log(options);
```

Comment:
It should return a middleware handler, for example.

return function checkRouteACLs(req, res, next) {...};